tensorflow/lite/micro/tools/project_generation/create_tflm_tree.py

```diff
@@ -179,6 +179,13 @@ def _create_examples_tree(prefix_dir, examples_list):
         # in-place find and replace.
         print(line, end="")
 
+def _rename_cc_to_cpp (output_dir):
+  for path, subdirs, files in os.walk(output_dir):
+    for name in files:
+      if fnmatch(name, "*.cc"):
+        baseName, extension = os.path.splitext(name)
+        os.rename (path + '/' + name, path + '/' + baseName + ".cpp")
+
```

Comment:
Find all *.cc files in output_dir and then rename them to baseName.cpp

Reply:
This whole block to find all of the files in a subdirectory came from this example: 
https://stackoverflow.com/a/13214966